tests/unit/store/playback.spec.js

```diff
@@ -124,71 +123,81 @@ describe("store/playback", () => {
+    test("When set_playback_state is committed, Then the playback_state is mutated", () => {
       store.commit(
         "playback/set_playback_state",
         playback_module.ENUMS.PLAYBACK_STATES.PLAYING
       );
-      expect(store.state.playback.playback_state).toEqual(
+      expect(store.state.playback.playback_state).toStrictEqual(
         playback_module.ENUMS.PLAYBACK_STATES.PLAYING
       );
       store.commit(
         "playback/set_playback_state",
         playback_module.ENUMS.PLAYBACK_STATES.STOPPED
       );
-      expect(store.state.playback.playback_state).toEqual(
+      expect(store.state.playback.playback_state).toStrictEqual(
         playback_module.ENUMS.PLAYBACK_STATES.STOPPED
       );
     });
 
-    it("should set is_playing to false initially", () => {
-      expect(store.getters["playback/is_playing"]).toBe(false);
+    test("When the playback store is initialized, Then is_playing is set to false", () => {
+      expect(
+        store.state.playback.playback_state ===
+          playback_module.ENUMS.PLAYBACK_STATES.PLAYING
+      ).toBe(false);
     });
-    it("mutates is_playing", () => {
+    test("When playback_state is set to PLAYING and STOPPED, Then is_playing is true and false respectively", () => {
       store.commit(
         "playback/set_playback_state",
         playback_module.ENUMS.PLAYBACK_STATES.PLAYING
       );
-      expect(store.getters["playback/is_playing"]).toBe(true);
+
+      expect(
+        store.state.playback.playback_state ===
+          playback_module.ENUMS.PLAYBACK_STATES.PLAYING
+      ).toBe(true);
       store.commit(
         "playback/set_playback_state",
         playback_module.ENUMS.PLAYBACK_STATES.STOPPED
       );
-      expect(store.getters["playback/is_playing"]).toBe(false);
+      expect(
+        store.state.playback.playback_state ===
+          playback_module.ENUMS.PLAYBACK_STATES.PLAYING
+      ).toBe(false);
     });
 
-    it("should set loop_playback to false initially", () => {
+    test("When the playback store is initialized, Then loop_playback is set to false", () => {
       expect(store.getters["playback/loop_playback"]).toBe(false);
     });
 
-    it("mutates loop_playback", () => {
+    test("When the loop_playback mutates, Then value set is true or false", () => {
       store.commit("playback/set_loop_playback", true);
       expect(store.getters["playback/loop_playback"]).toBe(true);
       store.commit("playback/set_loop_playback", false);
       expect(store.getters["playback/loop_playback"]).toBe(false);
     });
 
-    it("should set x_time_index to 0 initially", () => {
+    test("When the playback store is initialized, Then the x_time_index is initially 0", () => {
       expect(store.getters["playback/x_time_index"]).toBe(0);
     });
-    it("mutates x_time_index", () => {
+    test("When the x_time_index mutates, Then the value set to 12345", () => {
```

Comment:
try not to include numbers in test titles unless they are specifically needed (for example testing the default value). Here you could say something like `When a value is committed to set_x_time_index, Then the x_time_index in the store gets updated to that value`